bin/convert-argv.js

```diff
@@ -80,12 +69,25 @@ module.exports = function(...args) {
 		const configArgList = Array.isArray(argv.config) ? argv.config : [argv.config];
 		configFiles = configArgList.map(mapConfigArg);
 	} else {
-		for (i = 0; i < defaultConfigFiles.length; i++) {
-			const webpackConfig = defaultConfigFiles[i].path;
-			if (fs.existsSync(webpackConfig)) {
+		const defaultConfigFileNames = ["webpack.config", "webpackfile"]
+			.map(function(filename) {
+				return extensions.map(function(ext) {
+					return filename.concat(ext);
+				});
+			})
+			.reduce(function(a, i) {
+				return a.concat(i);
+			}, []);
+
+		for (i = 0; i < defaultConfigFileNames.length; i++) {
```

Comment:
We will need some logic change here. Let's see compare both implementation.
**existing:** try to find file from  `["webpack.config", "webpackfile"]` in cwd.

**current-modification:** 
1. try to find file `webpack.config` in up-fashion.
2. if found then return, if not go to step#3
3. try to find file `webpackfile` in up-fashion.
4. if found then return

Do we see the problem here? Even though `webpackfile` is there one level above, it would not be discovered, but 2 level up `webpack.config` would be given chance first.  
I would want both files to get a fair chance to be discovered. @evenstensberg @ematipico your thoughts on this?

Reply:
top-down priority is the best

Reply:
@evenstensberg I did not get you. Can you give any example?

Reply:
@hemal7735 I think you are right. I implemented a solution with (dynamically created) RegExp, but I didn't merge it into this PR. I also added `webpackfile` related tests. Could you please check it? https://github.com/lakatostamas/webpack-cli/compare/1f2b7e2..727a59a

Reply:
Yeah both files should always have the same priority at each path of directory structure.